final_prep/annotations.rb

```diff
@@ -16,28 +23,46 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  #returns the built_bear hash
   return built_bear
 end
-
+#calls the build_a_bear method with arguments "Fluffy, 4, brown, an array of
+#pants, jorts, and a tanktop, and 'give you nightmares'"
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
+#calls the build_a_bear method with arguments "sleepy", 2, purple, an array of
+#pajamas and a sleeping cap, and sleeping in
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
 
+#creats a method named fizzbuzz calling for arguements num_1, num_2, and range
```

Comment:
These are called _parameters_,.Arguments are the values that you actually pass into the method (i.e. on lines 66 and 68). Parameters and arguments are 2 sides of the same coin, so to speak.